src/pages/Admin/Venue/Rooms/RoomMapDropdown.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useMemo, useState, Dispatch, SetStateAction } from "react";
+import { useCallback } from "react";
+import { Dropdown, DropdownButton } from "react-bootstrap";
+
+const roomMapZoomLevels = [
+  { level: 1 },
+  { level: 2 },
+  { level: 3 },
+  { level: 4 },
+];
```

Comment:
move into settings. Why do we need to specify it? Let's also set the `DEFAULT_ZOOM_LEVEL`

Reply:
name the array `AVAILABLE_ZOOM_LEVELS = [1, 2, 3, 4]`

Reply:
fixed 8cc47b0